frontend/src/store/useFeedFilterStore.tsx

```diff
@@ -20,4 +21,5 @@ export const useFeedFilterStore = create<FilterState>((set) => ({
 
   // Actions
   setStatusFilter: (status) => set({ statusFilter: status }),
+  setSortOrder: (order: SortOrderType) => set({ sortOrder: order }),
```

Comment:
You'll notice that it will only sort the currently loaded submissions by the order

-- but we want to sort ALL submissions by order. So we should actually be passing this order all the way to the API so that we can fetch paginated results starting with the oldest items